types/stripe-v3/index.d.ts

```diff
@@ -165,6 +197,94 @@ declare namespace stripe {
         metadata: any;
         name?: string;
         tokenization_method?: tokenizationType;
+        three_d_secure?: 'required' | 'recommended' | 'optional' | 'not_supported';
+    }
+
+    interface RetrieveSourceOptions {
+        id: string;
+        client_secret: string;
+    }
+
+    // Container for all payment request related types
+    namespace paymentRequest {
```

Comment:
https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#namespace

I know the other ones do not follow that convention, it could be good to change them too

Reply:
I don't want to change the existing ones as that will break backwards compatibility. I wasn't sure if this namespace is necessary or not. Do you think it makes it a bit cumbersome to import the types? I could just not have an extra interface for the payment request stuff. What do you think?

Reply:
I like the namespace, it's true that the change will break backwards compatibility for existing namespaces, let's leave it in camelCase 👍